src/pages/guides/packages/venia/sample-data/index.md

```diff
@@ -1,54 +1,64 @@
 ---
 title: Venia sample data
+adobeio: /guides/packages/venia/sample-data/
 ---
 
-# Venia sample data
+![Accessories sample data](images/accessories-sample-data.png)
 
-The [Venia demo storefront][] runs on top of an Adobe Commerce backend that has a specific set of sample data installed.
-The sample Adobe Commerce backend available to developers also contains the same sample data set.
+The Venia storefront looks best when running against a Magento 2 backend with the Venia sample data installed.
 
-[venia demo storefront]: https://venia.magento.com/
+{: .bs-callout .bs-callout-info}
+The `builpack create-env-file` command defaults to the URL for a Magento 2 cloud instance that has the Venia sample data installed, so
+setting up a Magento 2 instance and installing sample data into it is now an optional step.
 
-![venia-sample-data](images/accessories-sample-data.png)
+Follow the instructions on this page to install the Venia sample data into your Magento 2 development instance.
 
-## Installing the Venia sample data
+## Prerequisites
 
-If you have access to your own Adobe Commerce or Magento Open Source backend and you want to install the Venia sample data, follow these instructions.
+* PHP 7.1.3+
+* System access to a Magento 2 instance
 
-### Step 1. Copy or create the deploy script
+{: .bs-callout .bs-callout-warning}
+If you have the previous `magento2-sample-data` module installed, you need to [remove the sample data modules][] and re-install Magento with a clean database.
 
-Copy the [deployVeniaSampleData.sh][] script into the root directory of your backend installation.
-A copy of this script is also included when you [setup your storefront project][].
+## Step 1. Set the composer repository in the config
 
-[deployveniasampledata.sh]: https://github.com/magento/pwa-studio/blob/develop/packages/venia-concept/deployVeniaSampleData.sh
-[setup your storefront project]: /tutorials/setup-storefront/
+Run the following command in the Magento root directory setup the repository for the sample data in https://repo.magento.com:
 
-### Step 2. Execute the deploy script
+```sh
+composer config --no-interaction --ansi repositories.venia-sample-data composer https://repo.magento.com
+```
+
+## Step 2. Require in the sample data
 
-Execute the script in the root directory of your application to add the Venia sample data modules to the backend application:
+Run the following command in the Magento root directory to update the composer.json to require in magento/venia-sample-data metapackage:
 
 ```sh
-bash deployVeniaSampleData.sh
+composer require --no-interaction --ansi magento/venia-sample-data:0.0.1
```

Comment:
would be good to mention that the version should be the latest released and not always be `0.0.1`